applications/ShapeOptimizationApplication/custom_utilities/geometry_utilities.h

```diff
@@ -333,6 +317,127 @@ class GeometryUtilities
     ///@name Private Operations
     ///@{
 
+    // --------------------------------------------------------------------------
+    void CalculateAreaNormalOnSimplex(ConditionsArrayType& rConditions, int dimension)
```

Comment:
The term "Simplex" is not correct anymore, because as far as i understand quads are not simplex geometries (only triangle/tetras...)

Reply:
True, I will change it.